src/commands/compress.rs

```diff
@@ -55,6 +55,10 @@ pub fn compress_files(
                 encoder,
                 level.map_or_else(Default::default, |l| bzip2::Compression::new((l as u32).clamp(1, 9))),
             )),
+            Bzip3 => Box::new(
+                // Unwrap is safe when using a valid block size
+                bzip3::write::Bz3Encoder::new(encoder, bytesize::ByteSize::mib(5).0 as usize).unwrap(),
```

Comment:
5 MiB sounds OK for now, although I'd prefer something around 10MB-20MB.

Right now we don't allow the user to provide compression levels for each format separately, something like `--levels bz=3,zst=7`, should we allow `.bz3` to accept a MB amount from the `--level` flag? It might be confusing with how `--level` is used by other formats.

@figsoda any thoughts?

Reply:
> should we allow `.bz3` to accept a MB amount from the `--level` flag? It might be confusing with how `--level` is used by other formats.

sounds good to me, this should be consistent with the `--block` flag of `bzip3`, assuming you mean `MiB`

alternatively we can just not respect the --level flag at all